interface/modules/custom_modules/oe-module-comlink-telehealth/src/Util/AuthUtils.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Comlink\OpenEMR\Modules\TeleHealthModule\Util;
+
+class AuthUtils
```

Comment:
note there is also a AuthUtils class in core OpenEMR, which could cause confusion

Reply:
It is namespaced, but I went ahead and renamed it to TelehealthAuthUtils to mitigate confusion.